mrbelvedereci/build/models.py

```diff
@@ -117,6 +117,7 @@ def run(self):
             self.time_end = timezone.now()
             self.save()
             self.delete_build_dir()
+            self.logger.handler.stream.flush(force=True)
```

Comment:
RE: self.logger.handler it seems it would be invalid but I confirmed this worked in my testing…